ts/WoltLabSuite/Core/Component/Quote/Message.ts

```
@@ -492,3 +498,42 @@ function removeSelection(): void {
     selection.removeAllRanges();
   }
 }
+
+function alignQuoteButtons(content: HTMLElement): void {
+  const coordinates = getElementBoundaries(window.getSelection())!;
```

What is the point of the not-null-assertion if there are cases in which this could indeed be `null`? Either you should handle this or `getElementBoundaries()` should throw if this is considered to be an invalid state.